### DISCUSSION TITLE
URL for Rest API for GitHub Enterprise server account

### DISCUSSION THREAD

**Discussion by himankgupta81** [community/community#24365] [Enterprise]
<p>If we are using an Enterprise Server account where we do not go to <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> but have our own customized URL, what would be the URL of the Rest API endpoint. will it be formed by appending /api/v3 in the end or can this be customized as well?</p>

**Comment by byrneh** [comment, marked as answer]
<p><a class="mention" href="/himankgupta81">@himankgupta81</a> correct append /api/v3 when using GHES, it is not configurable that I am aware</p>
